examples/cyclotron/run_cyclotron.py

```diff
@@ -49,21 +48,19 @@
 
 # design the accelerator lattice)
 ns = 25  # number of slices per ds in the element
-fodo = [
+period = [
     monitor,
-    elements.Drift(ds=0.25, nslice=ns),
-    monitor,
-    elements.Quad(ds=1.0, k=1.0, nslice=ns),
-    monitor,
-    elements.Drift(ds=0.5, nslice=ns),
-    monitor,
-    elements.Quad(ds=1.0, k=-1.0, nslice=ns),
-    monitor,
-    elements.Drift(ds=0.25, nslice=ns),
+    elements.ChrAcc(ds=0.038, ez=1.12188308693e-4, bz=1.0e-14, nslice=ns),
+    elements.ExactSbend(ds=0.25, phi=180.0, B=1),
+    elements.ChrAcc(ds=0.038, ez=1.12188308693e-4, bz=1.0e-14, nslice=ns),
+    elements.ExactSbend(ds=0.25, phi=180.0, B=1),
     monitor,
```

Comment:
See note in other input file: maybe I was not smart putting the monitor like this at the end, because it will be doubled every turn at the same location.